### PR TITLE
[FAI-13519] - GitHub Copilot usage recordedAt timestamp

### PR DESCRIPTION
**Pull request by crupley**
## Description

Populates the `recordedAt` field on `vcs_UserToolUsage` from the `emitted_at` timestamp on the source.

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

### CODE REVIEW

File: destinations/airbyte-faros-destination/test/converters/__snapshots__/faros_github.test.ts.snap [hunk 1]
```diff
@@ -34,6 +34,7 @@ Array [
   },
   Object {
     "vcs_UserToolUsage": Object {
+      "recordedAt": "2024-06-20T09:54:42.836Z",
```
Comment (crupley) [inline destinations/airbyte-faros-destination/test/converters/__snapshots__/faros_github.test.ts.snap:37]: I updated this manually to match the value in the records file; is there a better way?

File: destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts [hunk 2]
```diff
@@ -80,6 +80,7 @@ export class FarosCopilotSeats extends GitHubConverter {
         record: {
           userTool,
           usedAt: Utils.toDate(activeSeat.last_activity_at),
+          recordedAt: Utils.toDate(record.record.emitted_at),
```
Comment (crupley) [inline destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts:83]: Is it safe to use the `emitted_at` field from the stream here?

File: destinations/airbyte-faros-destination/test/converters/__snapshots__/faros_github.test.ts.snap (hunk 1 above)
Comment (dbruno21) [reply destinations/airbyte-faros-destination/test/converters/__snapshots__/faros_github.test.ts.snap:37]: Make sure to build before running converter tests:
`npm run build`

Run faros_github converter tests and pass the flag to update the snapshot:
`npm run test faros_github -- -u`

Comment (crupley) [reply destinations/airbyte-faros-destination/test/converters/__snapshots__/faros_github.test.ts.snap:37]: confirmed that `npm run test faros_github -- -u` yields the same snapshot file as what's here

File: destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts (hunk 2 above)
Comment (ypc-faros) [reply destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts:83]: You can safely assume that field is present (it's written by the CDK)
It represents the point in time at which the record was emitted by the source

**Review by ypc-faros — APPROVED** [review]

File: destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts (hunk 2 above)
Comment (crupley) [reply destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts:83]: That's exactly what I want to capture here. But maybe I should have asked: Is it safe to use a system field like this or would it be better to write a value from the source as part of the record `data` and pick that up in the destination?

Comment (ypc-faros) [reply destinations/airbyte-faros-destination/src/converters/github/faros_copilot_seats.ts:83]: I wouldn't bother if it's effectively $now

**Pull request merged**